skbio/io/format/blast6.py

```diff
@@ -0,0 +1,287 @@
+|nident     |Number of identical matches         |int  |
++-----------+------------------------------------+-----+
+|mismatch   |Number of mismatches                |int  |
++-----------+------------------------------------+-----+
+|positive   |Number of positive-scoring matches  |int  |
++-----------+------------------------------------+-----+
+|gapopen    |Number of gap openings              |int  |
++-----------+------------------------------------+-----+
+|gaps       |Total number of gaps                |int  |
++-----------+------------------------------------+-----+
+|ppos       |Percentage of positive-scoring matc\|float|
+|           |hes                                 |     |
++-----------+------------------------------------+-----+
+|frames     |Query and subject frames separated \|str  |
+|           |by a '/'                            |     |
++-----------+------------------------------------+-----+
+|qframe     |Query frame                         |int  |
++-----------+------------------------------------+-----+
+|sframe     |Subject frame                       |int  |
++-----------+------------------------------------+-----+
+|btop       |Blast traceback operations (BTOP)   |int  |
++-----------+------------------------------------+-----+
+|staxids    |Unique Subject Taxonomy ID(s), sepa\|str  |
+|           |rated by a ';' (in numerical order) |     |
++-----------+------------------------------------+-----+
+|sscinames  |Unique Subject Scientific Name(s), \|str  |
+|           |separated by a ';'                  |     |
++-----------+------------------------------------+-----+
+|scomnames  |Unique Subject Common Name(s), sepa\|str  |
+|           |rated by a ';'                      |     |
++-----------+------------------------------------+-----+
+|sblastnames|unique Subject Blast Name(s), separ\|str  |
+|           |ated by a ';' (in alphabetical\     |     |
+|           |order)                              |     |
++-----------+------------------------------------+-----+
+|sskingdoms |unique Subject Super Kingdom(s), se\|str  |
+|           |parated by a ';' (in alphabetical\  |     |
+|           |order)                              |     |
++-----------+------------------------------------+-----+
+|stitle     |Subject Title                       |str  |
++-----------+------------------------------------+-----+
+|sstrand    |Subject Strand                      |str  |
++-----------+------------------------------------+-----+
+|salltitles |All Subject Title(s), separated by \|str  |
+|           |a '<>'                              |     |
++-----------+------------------------------------+-----+
+|qcovs      |Query Coverage Per Subject          |int  |
++-----------+------------------------------------+-----+
+|qcovhsp    |Query Coverage Per HSP              |int  |
++-----------+------------------------------------+-----+
+
+List of data and descriptions taken from BLAST Command Line Applications User
+Manual [2]_.
+
```

Comment:
Can you add a Format Parameters section here describing `columns` and `default_columns` parameters? See the other I/O formats for examples of how we're formatting this section.